dkpro-core-castransformation-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/castransformation/ApplyChangesAnnotator.java

```diff
@@ -58,10 +53,6 @@
 {
     public static final String VIEW_SOURCE = "source";
     public static final String VIEW_TARGET = "target";
-    
-    public static final String OP_INSERT = "insert";
-    public static final String OP_REPLACE = "replace";
-    public static final String OP_DELETE = "delete";
```

Comment:
I need to re-establish those since that may break clients that reference these values, but I could reference the new constants and deprecate the old ones.

Reply:
This was just a suggestion when following the layering guidlines I am used to working with. I will revert this, it makes no functional difference.